### PR TITLE
Dictionary

### PR DESCRIPTION
**Pull request by EM0T10N**
Hey Steve,
Here's my pull Request!

### CODE REVIEW

File: todo-cli/main.py [hunk 1]
```diff
@@ -25,28 +25,46 @@ def print_welcome_message():
...
+def complete():
+    task_name = input('What task have you completed?')
+    task_status = "complete"
+    task = {
+        "name" : task_name,
+        "status": task_status
+    }
+    for task in todo_list:
+        if task["name"] == task_name:
+             todo_list.(task)
```
Comment (steve-curran) [inline todo-cli/main.py:65]: Here it looks like you are trying to add a new task to the list, but you are missing the method name on the `todo_list` item. If you wanted to add you would use `todo_list.append(task)` 

However, if you do that, you will have two tasks in the list with the same name, one with status set to `complete` from this function and one that was added by the `add` function. 

What you need to do is change the status of the item that is already in the list with the same name. 

You have already found that item using the `if` statement, now you just need to change its `status` property to `"complete"`

**Comment by EM0T10N** [comment]
I added anew commit and I think ( and hope) it uploaded onto here when I committed it

File: todo-cli/main.py [hunk 2]
```diff
@@ -25,28 +25,46 @@ def print_welcome_message():
...
+              return 
+
+    print("It's not in the list.")
         
+def complete():
+    task_name = input('What task have you completed?')
+    
+    for task in todo_list:
+        if task["name"] == task_name:
+             task["status"] = "complete"
```
Comment (steve-curran) [inline todo-cli/main.py:61]: Nice!! This works!

File: todo-cli/main.py [hunk 3]
```diff
@@ -25,28 +25,46 @@ def print_welcome_message():
...
+             print(f'Task "{task_name}" marked as complete.')
+             return
+        
+        print(f'Task"{task_name}" not found in the to-do list.')
```
Comment (steve-curran) [inline todo-cli/main.py:65]: This is causing a bug where the message prints out multiple times - can you guess how to fix it? 

To observe the bug - Add more than one item to the to-do list, then try to Complete an item that does not exist - you will see this message repeated for each item in the to-do list.

**Review by steve-curran — CHANGES_REQUESTED** [review]
Brilliant! 

Just one small bug to fix!

**Comment by EM0T10N** [comment]
I fixed the bug!

**Review by steve-curran — APPROVED** [review]
Nice one - You are good to merge now!

**Pull request merged**